.nvmrc

```diff
@@ -0,0 +1 @@
+stable
```

Comment:
There is a minimum version of node for NextJS and I was below minimum. With this file running `nvm use` fixes the situation and is generally safe.